config/environments/staging.php

```diff
@@ -0,0 +1,7 @@
+<?php
```

Comment:
I guess we can get rid of this environment now or change the dev to staging as you suggested.

Reply:
Good point.  I'll pull it.